services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -1576,7 +1587,7 @@ class ApplicationServiceITest : IntegrationTest() {
                 "startTime": "${nextYear()}-02-20T23:45:56Z",
                 "endTime": "${nextYear()}-02-21T00:12:34Z",
                 "pendingOnClient": true,
-                "workDescription": "Work description.",
+                "workDescription": "${ApplicationFactory.DEFAULT_WORK_DESCRIPTION}",
```

Comment:
Maybe static import for `DEFAULT_WORK_DESCRIPTION`?

Reply:
I changed this to static import.